services/QuillLMS/client/app/bundles/PremiumHub/components/dataExportTableAndFields.tsx

```diff
@@ -154,6 +154,22 @@ export const DataExportTableAndFields = ({ queryKey, searchCount, selectedGrades
     })
   }
 
+  function startCsvDownload() {
```

Comment:
What about using name as the backend endpoint?  (i.e. createCsvReportDownload)

I realize that there is inevitably drift with these sorts of things.